Failed forkchoice spec test

**Describe the bug**

`get_head_minimal.test.ts` forkchoice spec tests failed with

```
1) forkchoice get_head
       filtered_block_tree:

      AssertionError: expected 25 to equal 24
      + expected - actual

      -25
      +24
      
      at testFunction (test/spec/altair/fork_choice/get_head_minimal.test.ts:77:33)
      at Context.<anonymous> (/Users/tuyennguyen/Projects/workshop/lodestar/packages/spec-test-util/src/single.ts:127:22)
```

**Expected behavior**

I temporarily skip the test, need to fix it.

--- a/packages/fork-choice/src/protoArray/protoArray.ts
+++ b/packages/fork-choice/src/protoArray/protoArray.ts
@@ -432,15 +432,15 @@ export class ProtoArray {
   /**
    * This is the equivalent to the `filter_block_tree` function in the eth2 spec:
    *
-   * https://github.com/ethereum/eth2.0-specs/blob/v0.12.2/specs/phase0/fork-choice.md#filter_block_tree
+   * https://github.com/ethereum/eth2.0-specs/blob/v1.0.1/specs/phase0/fork-choice.md#filter_block_tree
    *
    * Any node that has a different finalized or justified epoch should not be viable for the
    * head.
    */
   nodeIsViableForHead(node: IProtoNode): boolean {
     return (
-      (node.justifiedEpoch === this.justifiedEpoch || node.justifiedEpoch === 0) &&
-      (node.finalizedEpoch === this.finalizedEpoch || node.finalizedEpoch === 0)
+      (node.justifiedEpoch === this.justifiedEpoch || this.justifiedEpoch === 0) &&
+      (node.finalizedEpoch === this.finalizedEpoch || this.finalizedEpoch === 0)
     );
   }
 

--- a/packages/spec-test-runner/test/spec/altair/fork_choice/get_head_minimal.test.ts
+++ b/packages/spec-test-runner/test/spec/altair/fork_choice/get_head_minimal.test.ts
@@ -134,8 +134,6 @@ describeDirectorySpecTest<IForkChoiceTestCase, void>(
       };
     },
     timeout: 10000,
-    // TODO: fix this
-    shouldSkip: (testCase, name) => name === "filtered_block_tree",
     // eslint-disable-next-line @typescript-eslint/no-empty-function
     expectFunc: () => {},
   }

